lib/pericope.rb

```diff
@@ -129,6 +129,7 @@ def normalizations
       @_normalizations ||= [
         [/(\d+)[".](\d+)/, '\1:\2'],        # 12"5 and 12.5 -> 12:5
         [/[–—]/,           '-'],            # convert em dash and en dash to -
+        [/(\d+)[.]\s+(\d+)/, '\1,\2'],      # 119:12. 14 to 119:12,14
```

Comment:
We shouldn't convert a `.` to a `,` with a normalization when the matcher defines `.` as a chapter-and-verse separator.

I'm pretty sure the bug is occurring because the first normalization is `/(\d+)[".](\d+)/` instead of `/(\d+)\s*[".]\s*(\d+)/` (which would agree with the matcher)